cpp/include/ray/api/serializer.h

```diff
@@ -30,8 +30,8 @@ class Serializer {
   }
 
   template <typename T>
-  static T Deserialize(const char *data, size_t size, size_t &off) {
-    msgpack::unpacked unpacked = msgpack::unpack(data, size, off);
+  static T Deserialize(const char *data, size_t size, size_t *off) {
+    msgpack::unpacked unpacked = msgpack::unpack(data, size, *off);
```

Comment:
Why modify this?

Reply:
There are two overload function:
```
static T Deserialize(const char *data, size_t size, size_t offset);
static T Deserialize(const char *data, size_t size, size_t &off);
```
When call one of it will case a distinguish compile error, use '*off' to reduce the compile error.